src/tasks/shellcheck/installer.ts

```diff
@@ -36,22 +37,23 @@ const installForMac = async () => {
         .tool('brew')
         .arg('install')
         .arg('shellcheck')
-        .exec();
+        .exec(<toolRunner.IExecOptions>{ silent: true });
```

Comment:
Before there was a lot of log output content relative to brew/brew updates that was not about ShellCheck and thus not relevant to users (snippet below for reference). This line change exclude that content and keeps the output focused on ShellCheck

```sh
Updating Homebrew...
==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations
==> Auto-updated Homebrew!
Updated 3 taps (caskroom/versions, homebrew/cask and homebrew/core).
==> New Formulae
gitleaks
go@1.11
operator-sdk
solr@7.7
==> Updated Formulae
adwaita-icon-theme
amazon-ecs-cli
ansible
atomist-cli
aws-sdk-cpp
binaryen
bitrise
bnd
chakra
clojure
cmake
conan
conserver
convox
crowdin
dnscrypt-proxy
dnsviz
easyengine
elasticsearch
exploitdb
flow
fn
folly
fontforge
fonttools
freetype
frugal
geocode-glib
gexiv2
gitbucket
glibmm
glooctl
gloox
gmic
gmsh
gnome-builder
go
godep
goreleaser
grafana
grv
gtksourceview3
gtksourceview4
hg-fast-export
hledger
homebank
istioctl
jasper
jenkins-lts
jsonrpc-glib
ktmpl
kubeprod
libdazzle
libdill
libetpan
libgpg-error
libosmium
libpointing
libssh2
libuv
libvirt
llvm
lsd
lynis
mesa
minio-mc
mitie
mkl-dnn
mpfr
nats-streaming-server
netdata
nicovideo-dl
nnn
node
openfortivpn
openrct2
openvdb
pandoc
pandoc-citeproc
php-code-sniffer
phpunit
plantuml
podofo
ppsspp
prototool
pulumi
pypy
qt
r
rbspy
re-flex
rebar3
redis
rtags
ruby
ruby-build
ruby@2.5
s3fs
scalariform
serverless
ship
skaffold
slackcat
solr
sqlcipher
swi-prolog
swiftformat
taskell
tbb
teleport
template-glib
terragrunt
thors-serializer
tmux-xpanes
tomcat
tomcat@7
topgrade
ucloud
vala
watchexec
yarn
yelp-tools
youtube-dl
==> Deleted Formulae
at-spi2-atk
at-spi2-core
go@1.4
go@1.8
monax
solr@5.5
solr@6.6
tmux-cssh
zxing-cpp
```